src/App.tsx

```diff
@@ -1,14 +1,36 @@
 /* eslint-disable max-len */
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
 
 import { TodoList } from './components/TodoList';
 import { TodoFilter } from './components/TodoFilter';
 import { TodoModal } from './components/TodoModal';
 import { Loader } from './components/Loader';
+import { getTodos } from './api';
+import { Todo } from './types/Todo';
 
 export const App: React.FC = () => {
+  const [todosList, setTodosList] = useState<Todo[]>([]);
+  const [selectedTodo, setSelectedTodo] = useState<number>(0);
```

Comment:
it's unnecessary, bc TS already knows about that, bc you put the initial value
```suggestion
  const [selectedTodo, setSelectedTodo] = useState(0);
```